aws/ecs/server_metrics_etl.tf

```diff
@@ -44,6 +45,7 @@ module "unmasked_server_metrics_etl" {
   sg_id                          = var.sg_id
   template_file                  = file("task-definitions/metrics.json")
   event_rule_schedule_expression = var.schedule_expression
+  metric_token_secret_arn        = aws_secretsmanager_secret.metrics_token.id
```

Comment:
Not needed for the server metrics task

Reply:
i assume this was a typo and you meant appstore-metrics_etl